helsingborg-stad/helsingborg-io-sls-api
# refactor(getapistatus): remove unused lambda

## Explain the changes you’ve made
Removed unused getApiStatus lambda

## Explain why these changes are made
The getApiStatus lambda has been replaced with getMessages lambda and its not in use anymore, hence it can be removed.

## How to test

1. Checkout this branch
2. run `sls deploy` within service folder for `service status-api`
3. Run the EKB application and check that the status request isn't failing
